### PR TITLE
get widget name on creation. otherwise it will always be the first one.

### PR DESCRIPTION
**Pull request by Tschanzt**
@maethu this should fix the behaviour when there are multiple widgets on the same page.

### CODE REVIEW

File: ftw/referencewidget/resources/refwidget.js [hunk 1]
```diff
@@ -12,7 +12,7 @@
       widget.button.on("click", openOverlay);
       $(window).one("resize", resize);
 
-      widget.name = widget.button.closest(".field").data("fieldname");
+      widget.name = "";
```
Comment (maethu) [inline ftw/referencewidget/resources/refwidget.js]: This breaks several things:
- wrong name on given values
- wrong name compared to new selected values.
- etc.

**Comment by Tschanzt** [comment]
@maethu can you take another look?

**Comment by maethu** [comment]
I fixed a js linting error on you branch... 

... and currently I'm testing, if I find something small I'm gonna push directly on your branch 😄

**Comment by maethu** [comment]
@Tschanzt Please make a new release 👍

**Pull request merged**